scaletest/terraform/README.md

```diff
@@ -32,9 +32,12 @@ project_id = "some_google_project_id"
 1. Run `coder_init.sh <coder_url>` to setup an initial user and a pre-configured Kubernetes
    template. It will also download the Coder CLI from the Coder instance locally.
 
-1. Do whatever you need to do with the Coder instance.
+1. Do whatever you need to do with the Coder instance:
 
-   > To run Coder commands against the instance, you can use `coder_shim.sh <command>`.
+   > Note: To run Coder commands against the instance, you can use `coder_shim.sh <command>`.
    > You don't need to run `coder login` yourself.
 
+   - To create workspaces, run `./coder_shim.sh scaletest create-workspaces --template="kubernetes" --count=N`
```

Comment:
I believe that at some point you may want to include also rich parameters as they cause more DB calls in total (worst case scenario).